tests/sentry/api/endpoints/test_relay_globalconfig_v4.py

```diff
@@ -26,6 +27,45 @@ def inner(version, global_):
     return inner
 
 
+@pytest.fixture
+def globalconfig_get_mock_config(monkeypatch):
+    monkeypatch.setattr(
+        "sentry.relay.globalconfig.get_global_config",
+        lambda *args, **kargs: {"global_mock_config": True},
+    )
+
+
+def test_global_config():
```

Comment:
I plan to get rid of this test after https://github.com/getsentry/team-ingest/issues/193 is completed. I think we can benefit from this test until that point.

Reply:
👍 I strongly recommend librelay-based validation tests, they can prevent incidents.